pyproject.toml

```diff
@@ -14,7 +14,10 @@ classifiers=[
     "License :: OSI Approved :: Apache Software License"
 ]
 dependencies = [
-    "caikit[runtime-grpc,runtime-http]>=0.16.0,<0.17.0",
+    "caikit[runtime-grpc,runtime-http,interfaces-vision]>=0.16.0,<0.17.0",
+    "transformers>=4.27.1,<5",
+    "torch>2.0,<3",
+    "timm>=0.9.5,<1"
```

Comment:
whats `timm` for?

Reply:
It's a deep learning library that's used for some computer vision models! Some models, e.g., DEtection TRansformer require it to run